ci/taos/config/kas-files/kas-YOUR-PROJECT-sample.yml

```diff
@@ -0,0 +1,18 @@
+#
+# This is an example of kas yaml config
+# User should set proper yaml config file
+# NOTE that the value of url should be "PATH_WILL_BE_REPLACED_BY_CI".
+# The TAOS-CI replace this string with proper path for submitted PR
+#
+# REF: https://github.com/siemens/kas
```

Comment:
@helloahn , Could you add info on KAS for maintenance?:) For example, 

* before :  REF: https://github.com/siemens/kas 
* after    :  Reference - https://github.com/siemens/kas (Setup tool for bitbake based projects)

Reply:
Thanks for your suggestion! I updated the file. Please check.